### PR TITLE
[MWPW-139314] : Action-item Block tests scripts

### PR DESCRIPTION
**Pull request by skumar09**
[MWPW-139314](https://jira.corp.adobe.com/browse/MWPW-139314 ): Action-item Block tests scripts

PR includes 
- Automation test scripts for the following features of the Action-item block

1.    Action-item (small)
2.    Action-item (medium)
3.    Action-item (large)
4.    Action-item (center)
5.    Action-item (rounded)
6.    Action-item (float button)
7.    Action-item (scroller)
8.    Action-item (scroller with navigation)

- Updates to slack test run summary message
- Update .yaml's for slack webhook

### CODE REVIEW

File: global.setup.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ async function globalSetup() {
 
   // Check if the code is running in a GitHub CI/CD environment
   if (process.env.GITHUB_ACTIONS === 'true') {
-    console.info('----Executing Tests in GitHub ---------');
+    console.info('---- Running Tests in the GitHub ---------');
```
Comment (amauch-adobe) [inline global.setup.js:18]: to follow same text construct probably should say "----Running Test in the GitHub environment --------'

**Review by amauch-adobe — APPROVED** [review]
Approved, but see my small recommendation.

File: .github/workflows/debug.yml [hunk 2]
```diff
@@ -24,10 +24,11 @@ jobs:
         uses: ./
         env:
           labels: ${{ inputs.tags }}
-          reporter: html
+          reporter: html
```
Comment (Dli3) [inline .github/workflows/debug.yml:27]: Extra space here. @skumar09

File: selectors/milo/actionitem.block.page.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+export default class ActionItem {
+  constructor(page, type, nth = 0) {
```
Comment (JackySun9) [inline selectors/milo/actionitem.block.page.js:2]: no use for type

File: selectors/milo/actionitem.block.page.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    this.navigationPrevious = this.actionScroller.locator('.nav-grad.previous');
+    this.navigationNext = this.actionScroller.locator('.nav-grad.next');
+    this.nextButton = this.navigationNext.locator('.nav-button.next-button');
+    this.previousButton = this.navigationPrevious.locator('.nav-button.previous-button');
+
+    this.scrollerActionItems = this.scroller.locator('.action-item');
+
+    this.mainImage = this.actionItem.locator('.main-image').nth(nth);
+    this.mainImageDark = this.actionItem.locator('.main-image.dark').nth(nth);
+    this.image = this.mainImage.locator('img').nth(0);
```
Comment (JackySun9) [inline selectors/milo/actionitem.block.page.js:28]: 0 or nth?

File: utils/reporters/base-reporter.js [hunk 5]
```diff
@@ -91,15 +91,25 @@ class BaseReporter {
...
-    } else {
+      const repo = process.env.GITHUB_REPOSITORY;
+      const runId = process.env.GITHUB_RUN_ID;
+      runUrl = `https://github.com/${repo}/actions/runs/${runId}`; 
+    }else if (process.env.CIRCLECI) {
+      envURL = process.env.PR_BRANCH_LIVE_URL || 'N/A';
+      exeEnv = 'CircleCI Environment';
+      const workflowId = process.env.CIRCLE_WORKFLOW_ID;
+      const jobNumber = process.env.CIRCLE_BUILD_NUM;
+      runUrl = `https://app.circleci.com/pipelines/github/wcms/nala/10/workflows/${workflowId}/jobs/${jobNumber}`;
```
Comment (JackySun9) [inline utils/reporters/base-reporter.js:111]: is this URL correct?

**Review by JackySun9 — CHANGES_REQUESTED** [review]

File: selectors/milo/actionitem.block.page.js (hunk 4 above)
Comment (skumar09) [reply selectors/milo/actionitem.block.page.js:28]: it's 0 ( not nth for img)

File: utils/reporters/base-reporter.js (hunk 5 above)
Comment (skumar09) [reply utils/reporters/base-reporter.js]: - good catch, (updated the URL...)
-

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**